### PR TITLE
fix: tests list returns empty records

### LINKED ISSUE #742: Tests list returning empty records
When listing tests by API there is empty entries on the beginning of the list

### PR DESCRIPTION
**Pull request by exu**
## Pull request description 

closes #742 

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: internal/app/api/v1/tests.go [hunk 1]
```diff
@@ -86,6 +87,8 @@ func (s TestKubeAPI) ListTestsHandler() fiber.Handler {
 			}
 		}
 
+		fmt.Printf("%+v\n", len(crTests.Items))
+
```
Comment (vsukhin) [inline internal/app/api/v1/tests.go:91]: Is this debugging printing?

File: cmd/kubectl-testkube/commands/tests/create.go [hunk 2]
```diff
@@ -61,7 +63,8 @@ func NewCreateTestsCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&file, "file", "f", "", "JSON test file - will be read from stdin if not specified, look at testkube.TestUpsertRequest")
-	cmd.Flags().StringSliceVar(&tags, "tags", nil, "--tags 1,2,3")
+	cmd.Flags().StringVar(&name, "name", "", "Set/Override test name")
+	cmd.Flags().StringSliceVar(&tags, "tags", nil, "--tags tag1,tag2,tag3")
```
Comment (vsukhin) [inline cmd/kubectl-testkube/commands/tests/create.go:67]: Shouldn't it be like in other places `comma separated list of tags: --tags tag1,tag2,tag3"`?

**Review by vsukhin — COMMENTED** [review]
Looks good, couple minor notes

File: cmd/kubectl-testkube/commands/tests/create.go (hunk 2 above)
Comment (exu) [reply cmd/kubectl-testkube/commands/tests/create.go]: yes - good point - was doing find/replace and missed this one - done @vsukhin

File: internal/app/api/v1/tests.go (hunk 1 above)
Comment (exu) [reply internal/app/api/v1/tests.go]: good eye :) fixed

**Review by vsukhin — APPROVED** [review]
lgtm

**Pull request merged**